packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```diff
@@ -102,33 +97,36 @@ mkdir -p %{buildroot}%{_root_mandir}/man1
 mv %{buildroot}%{gem_instdir}/man/hammer.1.gz %{buildroot}%{_root_mandir}/man1/
 rm -f %{buildroot}%{gem_instdir}/man/*.asciidoc
 
-mkdir -p %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d
-install -m 644 .%{gem_instdir}/config/cli_config.template.yml \
-               %{buildroot}%{_root_sysconfdir}/%{confdir}/cli_config.yml
+mkdir -p %{buildroot}%{hammer_confdir}/cli.modules.d
+install -m 0644 .%{gem_instdir}/config/cli_config.template.yml \
+                %{buildroot}%{hammer_confdir}/cli_config.yml
 
 %files
 %dir %{gem_instdir}
 %{_root_bindir}/hammer
 %{_root_bindir}/hammer-complete
 %doc %{_root_mandir}/man1/hammer.1.gz
 %{_root_sysconfdir}/bash_completion.d/%{gem_name}
-%{_root_sysconfdir}/%{confdir}/cli.modules.d
-%config(noreplace) %{_root_sysconfdir}/%{confdir}/cli_config.yml
+%license %{gem_instdir}/LICENSE
 %{gem_instdir}/bin
-%{gem_instdir}/lib
+%{gem_libdir}
 %{gem_instdir}/locale
-%license %{gem_instdir}/LICENSE
 %exclude %{gem_cache}
 %{gem_spec}
+%{hammer_confdir}/cli.modules.d
+%config %{hammer_confdir}/cli_config.yml
```

Comment:
Looks like this is true for all, but does the `(noreplace)` part make sense to keep and include in the template? Are we consistent with all hammer (plugins) now?

Reply:
When this is merged, we're consistent with all plugins not having "noreplace"